cli/tests/pcluster/validators/test_directory_service_validators.py

```diff
@@ -111,6 +114,49 @@ def test_ldap_tls_reqcert_validator(ldap_tls_reqcert, expected_message):
     assert_failure_messages(actual_failures, expected_message)
 
 
+@pytest.mark.parametrize(
+    "password_secret_arn, error_from_secrets_manager, expected_message, expected_failure_level",
+    [
+        (
+            "arn:PARTITION:secretsmanager:REGION:ACCOUNT:secret:NOT_ACCESSIBLE_SECRET",
+            "ResourceNotFoundExceptionSecrets",
+            "The secret arn:PARTITION:secretsmanager:REGION:ACCOUNT:secret:NOT_ACCESSIBLE_SECRET does not exist.",
+            FailureLevel.ERROR,
+        ),
+        (
+            "arn:PARTITION:secretsmanager:REGION:ACCOUNT:secret:ANY_SECRET",
+            "AccessDeniedException",
+            "Cannot validate secret arn:PARTITION:secretsmanager:REGION:ACCOUNT:secret:ANY_SECRET "
```

Comment:
Up to you: if you want to simplify the parameters you can use `.*` in the `expected_message`.

Reply:
Thanks for the hint. Actually, I prefer the exact match